test/CMakeLists.txt

```diff
@@ -87,17 +87,20 @@ endif()
 
 if( iodaconv_bufr_ENABLED )
   list( APPEND test_input
+    testinput/bufr_tables
     testinput/gdas.t18z.1bmhs.tm00.bufr_d
     testinput/gdas.t00z.1bhrs4.tm00.bufr_d
     testinput/gdas.t06z.adpsfc.tm00.bufr_d
     testinput/bufr_read_2_dim_blocks.bufr
+    testinput/20210203.radiosonde
```

Comment:
This is just YYYYMMDD, is this for a particular hour/cycle or the full day?

Reply:
where did this file come from?
WMO?  If so, we should atleast add something in the filename string a source of the data.

Reply:
ok, greg sent me this file a little while ago.

Reply:
For my feature branch doing more tests of WMO-style BUFR files, I am using names like this:
  synop_wmo_single.bufr
in which a single BUFR obs of a SYNOP is contained in the sample file.  So you could consider making yours "sonde_wmo_single.bufr" which I probably already have such a file in my branch.

In my own future work, then I plan to make 'multiple' replace single and change WMO reference to prepBUFR or something similar to handle next phases of this work.